io/post_fv3.F90

```diff
@@ -2996,29 +2996,29 @@ subroutine set_postvars_fv3(wrt_int_state,mpicomp,setvar_atmfile,   &
 
             ! model level cloud fraction
             if(imp_physics == 11) then !GFDL MP
-            if(trim(fieldname)=='cld_amt') then
```

Comment:
Is it possible to replace the conditions 
       if(imp_physics == 11) then !GFDL MP
            if(trim(fieldname)=='cld_amt') then
            end if
       else
            if(trim(fieldname)=='cldfra') then
            end if
       end if
        with
       if(trim(fieldname)=='cld_amt' .or trim(fieldname)=='cldfra') then
       end if

Reply:
I think's it's good not doing that as for GFDLMP you can output both "cld_amt" and "cldfra" in the diag_table, and the cfr will end up using cldfra which is not recommended for GFDLMP.

Reply:
Ok, I see.